app/services/out_of_band_session_accessor.rb

```diff
@@ -51,12 +51,20 @@ def destroy
     )
   end
 
+  # @api private
+  # Only used for convenience in tests
+  def put_empty_user_session(expiration = 5.minutes)
+    data = { test_data: true }
+    put(data, expiration)
+  end
+
   # @api private
   # Only used for convenience in tests
   # @param [Pii::Attributes] pii
-  def put_pii(pii, expiration = 5.minutes)
+  def put_pii(profile_id, pii, expiration = 5.minutes)
```

Comment:
Next time I would convert these to keyword arugments, since `pii` used to be first and now it's not